src/scope-hoisting/hoist.js

```diff
@@ -112,6 +112,13 @@ module.exports = {
       });
 
       path.scope.setData('shouldWrap', shouldWrap);
+
+      if (!shouldWrap) {
```

Comment:
I referenced the binding here to not have to recrawl after the `scope.push` line 163